README.md

```diff
@@ -43,5 +43,5 @@ $ go test -v ./...
 Starting test server:
 
 ```
-$ go run server.go
+$ go run cmd/artifact.go
```

Comment:
Right, I wouldn't want the file to be `./cmd/main.go`, because that will generate a binary named `cmd`.  Naming it `./cmd/travis-artifacts/{whatever}.go` will result in a `travis-artifacts` binary, which is what I'd expect 👍.